netbox/dcim/filtersets.py

```diff
@@ -1479,7 +1483,7 @@ class ConsolePortFilterSet(
 
     class Meta:
         model = ConsolePort
-        fields = ('id', 'name', 'label', 'speed', 'description', 'mark_connected', 'cable_end')
+        fields = ('id', 'name', 'label', 'speed', 'description', 'mark_connected', 'cable_end', 'device_status')
```

Comment:
Filters defined on the filterset class don't need to be added here; `fields` just denotes the model fields for which filters should be generated automatically.